sdks/python/Makefile

```diff
@@ -22,6 +22,7 @@ generate:
 		sed 's/io.k8s.api.core.v1.//' | \
 		sed 's/io.k8s.apimachinery.pkg.apis.meta.v1.//' \
 		> $(WD)/swagger.json
+	cp ../../LICENSE $(WD)/LICENSE
```

Comment:
Could you check if this is being copied to be right path and that https://github.com/argoproj/argo-workflows/issues/7881 is resolved after this?

Reply:
I have run `make --directory sdks/python publish -B` to produce the `argo-workflows.tar.gz` and it does now contain the license file at the root directory, as it did in versions pre 6.0.0. This will resolve #7881.

Reply:
Thank you!